src/components/UI/__test__/Card.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import { Card } from 'components/UI/Card';
+
+const content = <p>Test</p>;
+
+describe('Card component', () => {
+  test('should render correct content with correct styles', () => {
+    render(<Card children={content} />);
```

Comment:
```suggestion
    render(<Card>{content}</Card>);
```

it's nice that you have this component tested

although in real-life projects there's no need to write tests for something that only renders it's own children 😄

Reply:
Thank u 😄

I will know for future. Should I delete that one now or can I leave it her for that demo/no-real app?

Reply:
you can keep it for now